nt-app/src/store/index.js

```diff
@@ -36,6 +36,7 @@ export const flagInfo = {
     0: {
         "NT_sync_perks":                  { name: "Share all perks", tooltip: "When grabbing perks the whole team will also get them." },
         "NT_team_perks":                  { name: "Team Perks", tooltip: "When grabbing certain perks (not all) the whole team will also get them." },
+        "NT_alt_heartache":               { name: "Change Heartache", tooltip: "Change Heartache to effect HP instead of Max HP."},
```

Comment:
```suggestion
        "NT_alt_heartache":               { name: "Change Heartache", tooltip: "Change Heartache to affect HP instead of Max HP."},
```

Reply:
Oh grammar... and after I made sure I used the correct one for the test cases.